posts/2022-03-31-nexus-conf.adoc

```diff
@@ -63,7 +61,16 @@ Java Champion https://twitter.com/emilyfhjiang[Emily Jiang's] is going to bring
 
 === Join us
 
-All conference attendees are invited to join us at the IBM booth where you can check out our new Space Rover demo. IBM employees will also be available at the Eclipse Foundation booth.
+Key activities
+
```

Comment:
I think this section might benefit from the definition list syntax, which gives you some special heading styles on your list items- see suggestions

Reply:
agree